src/dotnet/commands/dotnet-pack/NuGet/ManifestVersionUtility.cs

```diff
@@ -15,9 +15,6 @@ internal static class ManifestVersionUtility
         public const int TargetFrameworkSupportForDependencyContentsAndToolsVersion = 4;
         public const int TargetFrameworkSupportForReferencesVersion = 5;
         public const int XdtTransformationVersion = 6;
-        // Note that this version should change from 7 to 8 when the PackageType
-        // schema is merged into here
-        public const int ServiceableVersion = 7;
 
         public static int GetManifestVersion(ManifestMetadata metadata)
         {
```

Comment:
Can you diff this file between rel/1.0.0 and rel/1.0.0-preview2?  They should be identical.  That will tell us if we missed any more.